### DISCUSSION TITLE
What about privacy?

### DISCUSSION THREAD

**Discussion by 9acca9** [danny-avila/LibreChat#2508] [General]
Hi.
I dont find this. What happens with the interaction? It is private? or is used? I mean, everything i ask, how travel? it is saved in other place?
I dont speak english, that is why i ask this in multiple forms but i think that you could understand what im talking about.

Thank you very much!

**Comment by fuegovic** [comment]
Hope this answers your question, the conversation/interaction are stored in your mongodb (database). This is what allows you to see your previous conversation history. When you converse with an AI, the request is sent to the API provider of said AI. What they do with it is out of our control, as they each have their own policies on the subject.

**Reply by 9acca9** [reply]
oh, ok. Thank you!

**Reply by DoS007** [reply]
@fuegovic Does the frontend (browser) or backend (server) send the request?

**Reply by danny-avila** [reply]
Backend

**Reply by DoS007** [reply]
@danny-avila So, theoretically using external mongodb for librechat hosted elswehere, the hoster of librechat could get the requests without connecting to mongodb himself? The requests themselves are probably encrypted/ssled/httpsed?
